examples/contact/CMakeLists.txt

```diff
@@ -12,13 +12,19 @@ if(TRIBOL_FOUND)
         twist.cpp
     )
 
-    foreach(filename ${arg_CONTACT_EXAMPLES_SOURCES})
+    foreach(filename ${CONTACT_EXAMPLES_SOURCES})
         get_filename_component(example_name ${filename} NAME_WE)
 
         blt_add_executable(NAME        contact_${example_name}
                            SOURCES     ${filename}
                            OUTPUT_DIR  ${EXAMPLE_OUTPUT_DIRECTORY}
                            DEPENDS_ON  serac_physics serac_mesh)
+
+        if(SERAC_ENABLE_HIP)
+            blt_add_target_compile_flags(TO contact_${example_name} FLAGS "-fgpu-rdc")
+            blt_add_target_link_flags(TO contact_${example_name} FLAGS "-fgpu-rdc")
+        endif()
```

Comment:
is rdc flag something that should be applied to all gpu examples? or is it specific to contact?